code/__DEFINES/atmospherics.dm

```diff
@@ -133,3 +133,7 @@
 
 //LAVALAND
 #define LAVALAND_EQUIPMENT_EFFECT_PRESSURE 50 //what pressure you have to be under to increase the effect of equipment meant for lavaland
+
+// Reactions
+#define N2O_DECOMPOSITION_MIN_ENERGY		1400
+#define N2O_DECOMPOSITION_ENERGY_RELEASED	200000
```

Comment:
Get out fusion we have a 142x energy efficiency rate, literally destroying the universe

*this isnt a review comment just me joking around with values*